content/docs/user-guide/components/reference/paintbrush/paintbrush.md

```diff
@@ -0,0 +1,169 @@
+
+| Mode | Description | Example 1 | Example 2 |
+| - | - | - | - |
+| (None) | (Input images used in smoothing) |![Sample smoothing input image.](/images/user-guide/components/reference/paintbrush/paintbrush-smooth-mode-input.png) |![Sample smoothing input image 2.](/images/user-guide/components/reference/paintbrush/paintbrush-smooth-mode-input2.png) |
+| Gaussian | Takes a weighted average of the NxN values using a Gaussian bell curve distribution function. This is the most common smoothing method used by paint programs. It provides a good balance of smoothing out noise and preserving details. |![Sample smoothing Gaussian image.](/images/user-guide/components/reference/paintbrush/paintbrush-smooth-mode-gaussian.png) |![Sample smoothing Gaussian image 2.](/images/user-guide/components/reference/paintbrush/paintbrush-smooth-mode-gaussian2.png) |
+| Mean | Takes an unweighted average of the NxN values. This method smooths out noise and is the method least likely to preserve sharp details. |![Sample smoothing mean image.](/images/user-guide/components/reference/paintbrush/paintbrush-smooth-mode-mean.png) |![Sample smoothing mean image 2.](/images/user-guide/components/reference/paintbrush/paintbrush-smooth-mode-mean2.png) |
+| Median | Takes the middle value from the sorted set of NxN values. This method preserves sharper edges than the other smoothing methods and can completely eliminate noise, but it also produces less smooth results in general and might introduce noticeable artifacts. |![Sample smoothing median image.](/images/user-guide/components/reference/paintbrush/paintbrush-smooth-mode-median.png) |![Sample smoothing median image 2.](/images/user-guide/components/reference/paintbrush/paintbrush-smooth-mode-median2.png) |
+
+#### Smoothing Radius
+
+**Smoothing Radius** is the number of values to use in each direction from the center value when computing the smoothed value. The smoothing methods use NxN values, where `N = (radius * 2) + 1`. A radius of 1 is a 3x3 set of values, a radius of 2 is a 5x5 set of values, a radius of 3 is a 7x7 set of values, etc.
+
+This number is expressed in "Value Space". In other words, it is describing the number of adjacent values to use in each direction regardless of how the values are mapped into world space.
+
+A larger smoothing radius will generally cause a smoother result across larger distances, but it also has an increased performance cost.
+
+The Smoothing Radius is separate from the Paint Brush Size. The Paint Brush Size controls how many values will get smoothed, and the Smoothing Radius controls how many adjacent values are used to smooth each value. Only values within the Paint Brush will get smoothed, but values outside of the Paint Brush might be used for calculating the smoothed results.
+
+## Hotkeys and CVars
+
+| Hotkey | Description |
+| - | - |
+| `[` | Decreases the Paint Brush Size |
```

Comment:
```suggestion
| **[** | Decreases the Paint Brush **Size**. |
```
1. Hotkeys are bolded.
2. Sidenote: This is an example of where bolding properties really shines; the user can be assured that this is going to affect the property's value directly.